app/views/collections/show.rb

```diff
@@ -102,6 +103,21 @@ def version
 
     private
 
+    def og_image
```

Comment:
do you mean only the og_image method? if I move og_image_from_promo and og_image_from_hero as well I won't have access to the view concern for hero_configs, unless I refactor that into a general concern, but that'll mean refactoring the portal index view too. Do you think it's worth doing so?